src/Options.py

```diff
@@ -15,6 +15,7 @@ class FillerTrapPercent(Range):
 for category in category_table:
     for option_name in category_table[category].get("yaml_option", []):
         if option_name not in manual_options:
-            manual_options[option_name] = type(option_name, (Toggle,), {"default": True})
+            manual_options[option_name] = type(option_name, (DefaultOnToggle,), {"default": True})
+            manual_options[option_name].__doc__ = "Should items/locations linked to this option be enabled?"
```

Comment:
small bonus: AP's unit test told me docs are required for all options.
also since the default is true if we want it to be the same on the webhost it need to be a DefaultOnToggle instead

Reply:
Something simple we could do is allow creating off by default options,
by checking for some prefix char like we do for category we could generate options set to false by default.
On my local version I implemented it and tested and it work
```python
for option_name in category_table[category].get("yaml_option", []):
    option_default_off = option_name.startswith('!')
    if option_default_off:
        option_name = option_name[1:]
    if option_name not in manual_options:
        if option_default_off:
            manual_options[option_name] = type(option_name, (Toggle,), {"default": False})
        else: 
            manual_options[option_name] = type(option_name, (DefaultOnToggle,), {"default": True})
        manual_options[option_name].__doc__ = "Should items/locations linked to this option be enabled?"
```

Reply:
I think we should wait to do more with options until we design a proper options.json. The categories stuff aims for a simple way for people to define those with a simple effect, but this is basically the categories problem all over again... in that, to have space to properly allow expanding how people can customize this, we need to separate options into their own json as well.

The prefix stuff was only used in requires because expanding those would've been cumbersome and hard to parse. In general, we should prefer verbose ways to express things, but requires is an exception.